### PR TITLE
Release Checklist PR Auto-Comment

### PR DESCRIPTION
**Pull request by ServiceDog**
Task details: https://app.dework.xyz/arrow-air/services-1?taskId=fb0de7f9-d04a-491e-9316-c506ea238c59

Proof of operation here: https://github.com/Arrow-air/svc-devops-test/pull/34#issuecomment-1767112419

### CODE REVIEW

File: src/repositories_rust.tf [hunk 1]
```diff
@@ -274,6 +274,12 @@ locals {
       ".github/workflows/autosquash.yml" = {
         content = file("templates/rust-all/.github/workflows/autosquash.yml")
       }
+      ".github/workflows/release_checklist.yml" = {
+        content = file("templates/rust-all/.github/workflows/release_checklist.yml")
+      }
+      "scripts/release_checklist.py" = {
```
Comment (owlot) [inline src/repositories_rust.tf:280]: This should be added to the arrow-sanitychecks docker instead

File: src/templates/all/.make/rust.mk [hunk 2]
```diff
@@ -179,5 +179,9 @@ rust-ut-coverage: check-cargo-registry rust-docker-pull
 		--output-dir coverage/)
 	@sed -e "s/\/usr\/src\/app\///g" -i coverage/lcov.info
 
+release-checklist: check-cargo-registry rust-docker-pull
```
Comment (owlot) [inline src/templates/all/.make/rust.mk:182]: Run this through our sanitychecks docker

File: src/repositories_rust.tf (hunk 1 above)
Comment (ServiceDog) [reply src/repositories_rust.tf]: https://github.com/Arrow-air/tf-github/pull/140/commits/4eb31b9268274ae2bca55fc48543d26edcf6324f

File: src/templates/all/.make/rust.mk (hunk 2 above)
Comment (ServiceDog) [reply src/templates/all/.make/rust.mk]: https://github.com/Arrow-air/tf-github/pull/140/commits/4eb31b9268274ae2bca55fc48543d26edcf6324f

**Comment by ServiceDog** [comment]
PR to add release checklist script to sanity checks docker: https://github.com/Arrow-air/tools/pull/39

**Review by owlot — APPROVED** [review]

**Pull request merged**